packages/story-editor/src/types.js

```diff
@@ -67,6 +67,20 @@ StoryPropTypes.story = PropTypes.shape({
   backgroundAudio: BackgroundAudioPropType,
 });
 
+StoryPropTypes.outputStory = PropTypes.shape({
```

Comment:
Hmm I think we can move that inline in `packages/story-editor/src/output/story.js`